sema/commons/rdf_ogm/object_graph_mapper.py

```
@@ -0,0 +1,148 @@
+                    subject,
+                    IRITemplate("rdfs:label").expand(self),
+                    IRITemplate(_label).expand(self)
+                ))
+
+        if property:
+            for predicate, object in property.items():
+                if not object:
+                    continue
+
+                # make sure predicate (k) is expanded
+                predicate = IRITemplate(predicate).expand(self)
```

Ruff (0.12.2)

Undefined name `IRITemplate`

(F821)